src/txmempool.h

```diff
@@ -35,6 +35,9 @@ extern CCriticalSection cs_main;
 /** Fake height value used in Coin to signify they are only in the memory pool (since 0.8) */
 static const uint32_t MEMPOOL_HEIGHT = 0x7FFFFFFF;
 
+// this value is 3/4 ripped from max block weight in consensus.h
+static const unsigned int MAX_REBROADCAST_WEIGHT = 3000000;
```

Comment:
Can you explain why we're not rebroadcasting a full block weight's worth? And maybe clean up grammar here.